api/conf/schema.json

```diff
@@ -3022,6 +3022,34 @@
 		"upstream_hash_vars_schema": {
 			"pattern": "^((uri|server_name|server_addr|request_uri|remote_port|remote_addr|query_string|host|hostname)|arg_[0-9a-zA-z_-]+)$",
 			"type": "string"
+		},
+		"system_config":{
```

Comment:
No need to add a schema for this, this file will only be exported by the APISIX control API. The model looks simple and we check it manually when we modify the data.

Reply:
Please keep the same opinion, I think both are fine, but according to the existing code style, it should be schema @bzp2010 @nic-chen

Reply:
> Please take a look at this issue, schema.json will be automatically generated by control API only, if we add it manually, it will be difficult to maintain. Do you have any suggestion?

@jwrookie I think the comment only needs to be `resolved` by the reviewer himself, if it is always closed by your, other reviewers will not be aware of the progress of the review and existing problems. This time I didn't notice the content of this review. 😂

@nic-chen Please take a look at this issue, `schema.json` will be automatically generated by control API only, if we add it manually, it will be difficult to maintain. Do you have any suggestion?

Reply:
@bzp2010  
good catch!
we could put the schema into another file, and when the Manager API starts, we need to merge its content into `schema.json`

Reply:
I would be happy to do it by other pr